### PR TITLE
Add obs build script and spec file for opensuse

### PR DESCRIPTION
**Pull request by siteshwar**
Related: #520

### CODE REVIEW

File: packaging/opensuse/ksh.spec.in [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+AT&T Bell Laboratories.
+KornShell is a shell programming language, which is upward compatible
+with "sh" (the Bourne Shell).
+
+%prep
+%autosetup -n ast-%{commit}
+
+%build
+%meson -Dbuild-pty=false
+%meson_build
```
Comment (krader1961) [inline packaging/opensuse/ksh.spec.in]: Out of curiosity what does this line, and `meson_install` do? I don't see any functions in this script with those names and those commands are provided by the Meson project AFAICT.

**Comment by krader1961** [comment]
LGTM

File: packaging/opensuse/ksh.spec.in (hunk 1 above)
Comment (siteshwar) [reply packaging/opensuse/ksh.spec.in]: These macros are provided by Meson and are documented [here](https://fedoraproject.org/wiki/Packaging:Meson#Available_Macros).

**Comment by siteshwar** [comment]
Merged through 11398511

**Pull request closed without merging**